userspace/falco/configuration.cpp

```diff
@@ -32,8 +32,9 @@ using namespace std;
 falco_configuration::falco_configuration()
 	: m_buffered_outputs(true),
 	  m_webserver_enabled(false),
-	  m_webserver_listen_port(8765),
+	  m_webserver_listen_port("8765"),
 	  m_webserver_k8s_audit_endpoint("/k8s_audit"),
+	  m_webserver_ssl_enabled(false),
```

Comment:
As you mentioned in the comments, I think it's a bit confusing to have to change two things to enable ssl (setting webserver.ssl_enabled *and* adding a 's' to the port definition). How about we do as you suggested and add 's' based on the value of webserver.ssl_enabled?

Reply:
If we assume `webserver.ssl_enabled: true` means any ports specified will be SSL, that means you won't be able to listen on a non-SSL port and a SSL port at the same time. Generally I can't think why you'd want the falco web service to listen on multiple ports for this use case, and best practice should be that the kubernetes audit events are sent encrypted in transit as they could have sensitive information. If you're ok with this limitation I'll make the change.

Reply:
change pushed